### ISSUE TITLE
Can't be installed on Ubuntu 16.10

### ISSUE THREAD

**Issue by Jop-V** [Aseman-Land/Cutegram#266]
Not from the PPA (missing Release file), not from the .deb (deprecated dependencies), and not with the compiler (`Unknown replace function: relative_path`).

**Comment by pmgouveia** [comment]
yeah it depends on gstreamer-0.10-plugins-ugly and gstreamer-0.10-plugins-bad and they don't exist on ubuntu 16.10.. wonder why..?
